en/core-libraries/caching.rst

```diff
@@ -383,7 +383,7 @@ You can greatly improve the performance of your application by putting results
 that infrequently change, or that are subject to heavy reads into the cache.
 A perfect example of this are the results from
 :php:meth:`Cake\\ORM\\Table::find()`. The Query object allows you to cache
-results using the ``cache()`` method. See the :ref:`caching-query-results` section
+results using the ``cache()`` method. See the :ref:`caching-loaded-results` section
```

Comment:
You can only link to headings that have named destinations in RST. The HTML output includes anchors on every title but you can't link to them unless the heading is preceded by a 'ref' target.

For example

```rst
.. _caching-loaded-results

Caching Loaded Results
------------------------
```

Is how you could make a heading linkable. This is a requirement of how sphinx/RST works as it outputs more than just HTML pages.